packages/chart/src/components/BarChart.tsx

```diff
@@ -135,10 +176,11 @@ export default function BarChart({ xScale, yScale, seriesScale, xMax, yMax, getX
 
   return (
     <ErrorBoundary component='BarChart'>
-      <Group left={config.runtime.yAxis.size}>
+      <Group left={config.runtime.yAxis.size ? parseInt(config.runtime.yAxis.size) : 66}>
```

Comment:
Is 66 a magic number here?
https://en.wikipedia.org/wiki/Magic_number_(programming)

This line should probably be reset.

Reply:
This was a red console error occurring not related to the ticket but easy to fix.
THe 66 came from some default setting on yAxis.size that I found somewhere else.
So yes I suppose it is a magic number since one would not know where it came from.
You want it to be a CONSTANT defined somewhere else?

Reply:
I'm removing the ternary entirely and just adding parseFloat which is more forgiving than parseInt. Not sure what type of values are going into there.  I tested it and do not get the red error related to setting value "left" on group.